### PR TITLE
add boolean properties in UserMenu component (issue 1402)

### PR DESCRIPTION
**Pull request by diegosanmartino**

### CODE REVIEW

File: web/client/components/security/UserMenu.jsx [hunk 1]
```diff
@@ -58,13 +64,25 @@ const UserMenu = React.createClass({
   },
   renderLoggedTools() {
       let DropDown = this.props.nav ? NavDropdown : DropdownButton;
+      let showAccountInfoItem = null;
```
Comment (mbarto) [inline web/client/components/security/UserMenu.jsx]: I think it's better using an array of items and push each enabled item in it

Comment (diegosanmartino) [inline web/client/components/security/UserMenu.jsx]: ok

File: web/client/components/security/UserMenu.jsx [hunk 2]
```diff
@@ -58,13 +64,21 @@ const UserMenu = React.createClass({
...
       let DropDown = this.props.nav ? NavDropdown : DropdownButton;
+      let itemArray = [];
+      if (this.props.showAccountInfo) {
+          itemArray.push(<MenuItem key="accountInfo" onClick={this.props.onShowAccountInfo}> <Glyphicon glyph="user" /><Message msgId="user.info"/></MenuItem>);
+      }
+      if (this.props.showPasswordChange) {
+          itemArray.push(<MenuItem key="passwordChange" onClick={this.props.onShowChangePassword}> <Glyphicon glyph="asterisk" /> <Message msgId="user.changePwd"/></MenuItem>);
+      }
+      if (this.props.showLogout) {
+          itemArray.push(<MenuItem key="divider" divider />);
```
Comment (mbarto) [inline web/client/components/security/UserMenu.jsx]: I would push the divider only if itemArray.length > 0

Comment (diegosanmartino) [inline web/client/components/security/UserMenu.jsx]: ok

**Pull request merged**